tests/utils/verification/volumeproperties.go

```diff
@@ -161,3 +163,52 @@ func GetAssociatedPolicyName(hostname string, volName string) (string, error) {
 	}
 	return op, err
 }
+
+// CheckVolumeInProcMounts - check if volume entry is present in docker host mount file
+// grep does the work. If entry isn't present, err is not nil (Exit status 1)
+func CheckVolumeInProcMounts(hostName, volName string) bool {
+	cmd := "grep -q " + volName + vm.MountFile
+	_, err := ssh.InvokeCommand(hostName, cmd)
+
+	if err != nil {
+		log.Printf("No entry for volume %s in %s.", volName, vm.MountFile)
+		return false
+	}
+
+	return true
+}
+
+// CheckRefCountInLogs - check the log record for expected refcount of the volume.
+// grep does the work. If refcount isn't the expected one, err is not nil (Exit status 1)
+func CheckRefCountInLogs(hostName, volName string, logLines, expectedRefCnt int) bool {
+	cmd := "tail -" + strconv.Itoa(logLines) + vm.LogFile + " | grep " + volName + " | grep -q refcount=" +
+		strconv.Itoa(expectedRefCnt)
+	_, err := ssh.InvokeCommand(hostName, cmd)
+
+	if err != nil {
+		log.Printf("Unable to find refcount for %s in logs.", volName)
+		return false
+	}
+
+	return true
+}
+
+// CheckRecoveryRecordInLogs - check the recovery record printed by plugin after restart
+// The log record must contain expected refcount
+func CheckRecoveryRecordInLogs(hostName, volName string, logLines, expectedRefCnt int) bool {
+	cmd := "tail -" + strconv.Itoa(logLines) + vm.LogFile + " | grep 'Volume name=" + volName +
+		"' | grep 'mounted=true'"
+	op, err := ssh.InvokeCommand(hostName, cmd)
+	expectedStr := "count=" + strconv.Itoa(expectedRefCnt)
+	if err != nil {
+		log.Printf("No recovery record for %s in logs.", volName)
+		return false
+	}
+
+	log.Printf("Recovery record is %s ", op)
+
+	if strings.Contains(op, expectedStr) != true {
+		return false
```

Comment:
minor: need to add logging statement for better debugging before returning false.